cudamapper/tests/Test_CudamapperOverlapperTriggered.cu

```diff
@@ -188,7 +188,7 @@ TEST(TestCudamapperOverlapperTriggerred, OneAchorNoOverlaps)
     cudautils::device_copy_n(anchors.data(), anchors.size(), anchors_d.data()); //H2D
 
     std::vector<Overlap> overlaps;
-    overlapper.get_overlaps(overlaps, anchors_d, test_index, test_index);
```

Comment:
Can you add tests for `Overlapper::update_read_names`?

Reply:
added a couple of tests.

Reply:
Can you please add them to all places where `overlapper.get_overlaps(overlaps, anchors_d, test_index, test_index);` got replaced by `overlapper.get_overlaps(overlaps, anchors_d);`?

Reply:
since update_read_names loops through overlaps, it made sense to add the test for it only to the cases where `overlaps.size() > 0`. Therefore you don't see everywhere.
Also, this update_read_names which before used to take place inside `overlapper.get_overlaps(overlaps, anchors_d, test_index, test_index)`, its output was never verifies, I mean before this PR we did not have any checks for updating query and target read names.